README.md

```diff
@@ -68,6 +68,66 @@ Contributions are what makes the open source community such an amazing place to
 - If you have any suggestions for the projects, like reporting a bug, improving UI, or improving the README.md file, feel free to **open an issue** to discuss it, or directly create a pull request with the necessary changes.
 - Please make sure you check your spelling and grammar.
 - Create individual PR for each suggestion.
+
```

Comment:
make a this changes in contribution.md another file is there.

Reply:
Ok